src/components/book.jsx

```diff
@@ -4,7 +4,7 @@ import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/bookSlice';
 
 function Book({
-  title, author, id, category,
+  title, author, id, category, progress,
 }) {
   const dispatch = useDispatch();
   // const books = useSelector((data) => data.books.books);
```

Comment:
- [ ] Please remove commented code line 10 from your code base. It's a bad practice to have commented (unused) code.

Reply:
done!